.github/workflows/medic-benchmarks.yml

```diff
@@ -82,10 +82,9 @@ jobs:
         --set publisher.rate=5
         --set camunda-platform.operate.enabled=true
         --set camunda-platform.operate.image.repository=gcr.io/zeebe-io/operate
-        --set camunda-platform.operate.image.tag=${{ needs.benchmark-data.outputs.operate }}    
-        --set camunda-platform.elasticsearch.volumeClaimTemplate.resources.requests.storage=128Gi
-        --set camunda-platform.retentionPolicy.enabled=false
-        --set retentionPolicy.enabled=true
+        --set camunda-platform.operate.image.tag=${{ needs.benchmark-data.outputs.operate }}
+        --set camunda-platform.elasticsearch.master.persistence.size=128Gi \
+        --set camunda-platform.zeebe.retention.minimumAge=1d \
```

Comment:
:question: I'm not sure what the required minimum age is for Operate, but I guess this should be plenty right? :sweat_smile:

Reply:
I put it really low, so we can delete data early https://github.com/zeebe-io/benchmark-helm/blob/main/charts/zeebe-benchmark/values.yaml#L359